README.md

```diff
@@ -454,5 +454,6 @@ Please refer to instructions mentioned in section [Supporting projects with a ja
 In case of a message like `Unable to update build.gradle.kts. Please update with these instructions` we are usually not able to find the build files because of: 
 - Either it is missing in the package.
 - It is renamed.
-To resolve it please follow along the doc to add these dependencies to your build file. After a successful sync, the plugin will start generating tests for your repository. 
+To resolve it please follow along the doc to add these dependencies to your build file. After a successful sync, the plugin will start generating tests for your repository.
+Use this Readme.md file to rectify the errors
```

Comment:
@Kavinkumar-04 -- what is the benefit of this line Use this Readme.md file to rectify the errors